uiowa/uiowa
# Add research areas filter to people list block settings

* Resolves #3812
* Resolves #3818

# How to test
* Review bug reproduction steps in #3818
* `blt ds --site sandbox.uiowa.edu`
* `drush @sandbox.local uli node/51/layout`
* To test that bug is fixed, follow similar steps, setting a breakpoint at line 229 in the ListBlock class. Observe that `$form['settings']['overrides']['exposed_filters']` doesn't contain an entry under the key `field_person_research_areas_target_id`. Also observe after script execution completes and the block form loads that the Research Areas filter field has a title showing.

## Story
As a content editor, I can set choose research areas as a filter to display people when adding or editing a people list block.
